lib/src/utils/commands_extension.dart

```diff
@@ -114,21 +129,43 @@ extension CommandsClientExtension on Client {
         threadLastEventId: args.threadLastEventId,
       );
     });
-    addCommand('dm', (CommandArgs args) async {
+    addCommand('dm', (args, stdout) async {
       final parts = args.msg.split(' ');
-      return await args.room.client.startDirectChat(
-        parts.first,
+      final mxid = parts.first;
+      if (!mxid.isValidMatrixId) {
+        throw CommandException('You must enter a valid mxid when using /dm');
+      }
+
+      final roomId = await args.client.startDirectChat(
+        mxid,
         enableEncryption: !parts.any((part) => part == '--no-encryption'),
       );
+      stdout?.write(
+        DefaultCommandOutput(
+          rooms: [roomId],
+          users: [mxid],
+        ).toString(),
+      );
+      return null;
     });
-    addCommand('create', (CommandArgs args) async {
+    addCommand('create', (args, stdout) async {
+      final groupName = args.msg.replaceFirst('--no-encryption', '').trim();
```

Comment:
can we also make sure that `groupName` is null when the result of `replaceFirst` followed by `trim` is an empty string? It would prevent sending a `m.room.name` event with an empty string when room name is not specified.